src/components/DpMultistepNav/DpMultistepNav.vue

```diff
@@ -7,7 +7,7 @@
       :disabled="idx > activeStep"
       @click="changeStep(idx)"
       class="btn--blank"
-      :aria-label="step.title ? Translator.trans(step.title) : Translator.trans(step.label)"
+      :aria-label="step.title ? step.title : step.label"
```

Comment:
(Why) Can we just remove this?

Reply:
Yes, we using this component in core like:
```
<dp-multistep-nav
          v-if="loggedIn === false && showHeader"
          @change-step="val => step = val"
          :active-step="step"
          :steps="[{
            label: Translator.trans('statement.yours'),
            icon: commentingIcon,
            title: Translator.trans('statement.modal.step.write')
          }, {
            label: Translator.trans('personal.data'),
            icon: 'fa-user',
            title: Translator.trans('statement.modal.step.personal.data')
          }, {
            label: Translator.trans('recheck'),
            icon: 'fa-check',
            title: Translator.trans('statement.modal.step.recheck')
    }]" />
```
Is that wrong?